scripts/net-tuning.sh

```
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+CONF_FILE="/etc/sysctl.d/99-high-throughput-network.conf"
+
+echo "[+] Writing sysctl configuration to $CONF_FILE"
+
+sudo tee "$CONF_FILE" >/dev/null <<'EOF'
```

Shellcheck (0.11.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)